pallets/runtime/develop/build.rs

```diff
@@ -1,9 +1,12 @@
 use wasm_builder_runner::WasmBuilder;
 
+const BUILDER_REPO: &str = "https://github.com/PolymathNetwork/substrate-wasm-builder.git";
+const BUILDER_REV: &str = "b0303c15b662dd768bfe808e3ed47ec65305fe2a";
```

Comment:
Would it make sense to move these to `polymesh-runtime-common`?

Reply:
@vkandy 
I cannot move them into `polymesh-runtime-common`, due to a weird error on WASM compilation, because it needs to add `polymesh-runtime-common` as `dev-dependency`.

Instead of that, I extracted constants and builder into a new crate: `polymesh-build-tool`.
Pls take a look on that.